PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalCheckoutRequest.kt

```diff
@@ -66,6 +66,8 @@ import org.json.JSONObject
  *
  * @property contactPreference Preference for the contact information section within the payment flow. Defaults to
  * [PayPalContactPreference.NO_CONTACT_INFORMATION] if not set.
+ *
+ * @property amountBreakdown Breakdown of items associated to the total cost
```

Comment:
Do we need docstrings for `recurringBillingDetails` and `recurringBillingPlanType` here too?